PWGHF/D2H/TableProducer/candidateCreatorCharmResoReduced.cxx

```diff
@@ -418,9 +427,62 @@ struct HfCandidateCreatorCharmResoReduced {
   }
   PROCESS_SWITCH(HfCandidateCreatorCharmResoReduced, processLambdaDminusWithMl, "Process LambdaDminus candidates with Ml info", false);
 
-}; // struct
+}; // struct HfCandidateCreatorCharmResoReduced
+
+struct HfCandidateCreatorCharmResoReducedExpressions {
+
+  Produces<aod::HfMcRecRedResos> rowResoMcRec;
+
+  HistogramRegistry registry{"registry"};
+
+  void init(InitContext const&)
+  {
+    const AxisSpec axisPt{(std::vector<double>)vecBinsPt, "#it{p}_{T} (GeV/#it{c})"};
+    registry.add("hMassMcMatched", "Reso MC candidates Matched with generate particle;m (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{200, 2.5, 2.7}, {axisPt}}});
+    registry.add("hMassMcMatchedIncomplete", "Reso MC candidates Matched with generate particle w. Invcomplete decay;m (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{400, 2.3, 2.7}, {axisPt}}});
```

Comment:
Why the mass binning is different compared to the matched one?
```suggestion
    registry.add("hMassMcMatchedIncomplete", "Reso MC candidates Matched with generate particle w. Incomplete decay;m (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{400, 2.3, 2.7}, {axisPt}}});
```

Reply:
I forgot to change it